EIPS/eip-7848.md

```diff
@@ -0,0 +1,122 @@
+
+When the block numbered `VOTING_WINDOW_END` is created, the upgrade is considered "activated" if `REQUIRED_APPROVALS` or more blocks during the voting window had set `referenceImplementationHash` to the value `REFERENCE_IMPLEMENTATION_HASH`; otherwise, the upgrade fails.
+
+Blocks created from `VOTING_WINDOW_END` + 1 onward will use the new software behaviors if and only if the upgrade was successful.
+
+For proof-of-work networks and other scenarios, it is possible that one fork activates the upgrade while another does not. In any case, blocks strictly greater than `VOTING_WINDOW_END` shall be created according to the behaviors of the software considering the outcome of the votes in the voting window which are the ancestors of that block.
+
+Note: Just because a upgrade was activated, this does not necessarily mean that the new software behaviors must generate block `VOTING_WINDOW_END` + 1 differently than the old software behaviors. Perhaps the new software behaviors will stipulate that only blocks at some later time will generate differently. Perhaps the new software behaviors include some other consensus change which does not change how blocks are created.
+
+## Rationale
+
+### Forking is no Longer Possible
+
+Since the merge, forking Ethereum Mainnet has become practically impossible. Validators stake valuable assets to participate in the network, so any rational validator will choose to upgrade only if they expect widespread adoption. If a validator expects 95% or more participants to upgrade, they should upgrade; if they expect only 5% or less, they should not. For intermediate cases, there is a threshold where a validator would rationally shut down (incurring a small penalty) rather than risk participating in the wrong network—which could result in slashing 100% of their staked ether (currently 16 Ether per share).
+
+Therefore, proper management of consensual upgrades is crucial.
+
+### Community Direction
+
+1. The Ethereum project and community do not have an official mission statement or vision. However, this proposal asserts that the Ethereum community would prefer Ethereum Mainnet to be a community-directed project.
+2. On-chain signaling of upgrades enables community direction in a way that is not possible today.
+
+### Decentralization
+
+1. Currently, upgrade notices are published on blog.ethereum.org.
+2. The ethereum.org website has opaque ownership; for example, domain ownership (via whois) is redacted for privacy, and the website footer does not list a legal publisher.
+3. An opaque organization unilaterally announcing upgrades to client software is antithetical to the concept of a community-directed project.
+
+Under this new upgrade process, it would appropriate for Ethereum Foundation to change the way it blogs about upgrades as follows:
+
+```diff
+- The Ethereum network will be undergoing a scheduled network upgrade.
++ Ethereum Foundation proposes that clients upgrade and asks you to act now.
+```
```

Comment:
EIPs are _not_ the place to discuss anything related to the Ethereum Foundation.

```suggestion
```

Reply:
This is relevant to the community. But sure, it does not need to be discussed in the EIP proper.